question/bank/managecategories/tests/behat/category_reorder.feature

```diff
@@ -0,0 +1,104 @@
+@qbank @qbank_managecategories @javascript
+Feature: A Teacher can reorder question categories
+  In order to change question category order
+  As a Teacher
+  I need to reorder them
+
+  Background:
+    Given the following "users" exist:
+        | username | firstname | lastname | email                |
+        | teacher1 | Teacher   | 1        | teacher1@example.com |
+    And the following "courses" exist:
+        | fullname | shortname | format |
+        | Course 1 | C1        | weeks  |
+    And the following "categories" exist:
+        | name  | category | idnumber |
+        | Cat 1 | 0        | CAT1     |
+    And the following "activities" exist:
+        | activity | course | name           |
+        | quiz     | C1     | Test quiz Q001 |
+    And the following "course enrolments" exist:
+        | user      | course | role           |
+        | teacher1  | C1     | editingteacher |
+    And the following "question categories" exist:
+        | contextlevel    | reference      | name              |
+        | Activity module | Test quiz Q001 | Quiz category     |
+        | Activity module | Test quiz Q001 | Quiz category 2   |
+        | Course          | C1             | Course category   |
+        | Course          | C1             | Course category 2 |
+        | Category        | CAT1           | Category cat      |
+        | System          | S1             | System category   |
+    And I am on the "Test quiz Q001" "quiz activity" page logged in as "admin"
+    And I navigate to "Question bank > Categories" in current page administration
+
+  @javascript
+  Scenario: Teacher cannot move or delete single category under context
+    When I click on "//a[contains(text(), 'Default for Category 1')]/../../div[@class='float-right']/div[@class='float-left']" "xpath_element"
```

Comment:
my brain hurts looking at this xpath, please create a generator and possibly pass the Text (eg Default for Category 1) if its a standard one which looks like for the rest of the tests